### ISSUE TITLE
Posting other options to cascading select options values

### ISSUE THREAD

**Issue by princenyeche** [princenyeche/Context-Field-Rebuilder#1]
It seems Jira can't accept other option values when posting. Although it returns a successful message when a request is posted,  however, the value is discarded. Need to find a better way around this problem for _create_back_cf_options()_

### **Adding more context**: 
This is referring to when you have a select cascading field value and you want to post the options to the custom field context, this doesn't work if the child option has a pre-existing value. i.e. we can't post the 2nd value.

Secondly, if the option exist but didn't have the first child value, we can't add the first child value. No error message is shown, it shows as successful but this is not the expected result.

### In summary, two things needs to be solved
1. Ability to post first child value even after the Parent option exist
2. Ability to post second child value to child(n)... value to the Parent option

- It looks like this is actually a bug on the API [ACJIRA-2058](https://ecosystem.atlassian.net/browse/ACJIRA-2058)

**Comment by princenyeche** [comment]
This Bug has been fixed.
